test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -635,8 +637,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 	})
 
 	Describe("with a windows agent pool", func() {
-		// TODO stabilize this test
-		/*It("should be able to deploy an iis webserver", func() {
+		It("should be able to deploy an iis webserver", func() {
 			if eng.HasWindowsAgents() {
 				r := rand.New(rand.NewSource(time.Now().UnixNano()))
 				deploymentName := fmt.Sprintf("iis-%s-%v", cfg.Name, r.Intn(99999))
```

Comment:
line 644 - s/1709-1803/

Reply:
we changed the default version to the latest - Windows Server version 1803. Containers are not cross-compatible without enabling Hyper-V isolation which is an alpha feature